.github/workflows/msbuild.yml

```diff
@@ -0,0 +1,31 @@
+name: MSBuild
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
The CircleCI config ran `dotnet` in a windows environment and `msbuild` on linux (debian).

Any reason for switching?

Really we just want to make sure anvil gets compiled once with .NET and once with Mono. It just happens the docker image for Mono was based off Linux.
https://github.com/mono/docker/blob/9293c0cddf31a3dc829fccc6f8e1eb507a91cd34/6.12.0.182/slim/Dockerfile

Reply:
Oh strange, I just noticed that every workflow example used `ubuntu-latest` so I did the same, but I thought msbuild needed windows - I can change it so both use `ubuntu-latest`? Or whatever arrangement you prefer.

Reply:
hmmm, leave them different I guess